### ISSUE TITLE
Listening on layer events

### ISSUE THREAD

**Issue by francois2metz** [soal/vue-mapbox#147]
Hi,

I would like to support the listening of events on a specific layer, the ones you can catch with `"on(type, layerId, listener)"`. It works when we add a specific layer, but not on the map.

I'm thinking of adding special events handlers, such as: `click-layerid`, `mousedown-layerid`, and so on... 

```vue
<template>
  <MglMap
    :map-style="..."
    @mouseover-poi="handleMouseOver"
    @click-poi="handleClick"
  ></MglMap>
</template>

<script>
export default {
  methods: {
    handleClick() {},
    handleMouseOver(e) { e.map.getCanvas().style.cursor = 'pointer';}
  }
}
</script>
```

What do you think ?

**Comment by vinayakkulkarni** [comment]
You already have those in this library.

Common layer evts: https://soal.github.io/vue-mapbox/api/Layers/#events

**Comment by francois2metz** [comment]
Yes I would have to add a layer, but they are already on the original source and style.
